en doc update (#3)
roxblnfk/yii-cycle: docs/en/reading-schema.md

## File-based schema

If you want to avoid annotations, you can describe a schema in a PHP file.

> Use `FromFileSchemaProvider` to load a schema:

Reviewer: ```suggestion
Use `FromFilesSchemaProvider` to load a schema:
```

```php
# config/common.php
[
    'yiisoft/yii-cycle' => [
        // ...
        'schema-providers' => [

>             \Yiisoft\Yii\Cycle\Schema\Provider\FromFileSchemaProvider::class => [

Reviewer: ```suggestion
            \Yiisoft\Yii\Cycle\Schema\Provider\FromFilesSchemaProvider::class => [
```

>                 'file' => '@runtime/schema.php'

Reviewer: ```suggestion
                'files' => '@runtime/schema.php'
```

Note that: 

> 1. `FromFileSchemaProvider` loads a schema from a PHP-file via `include`. That requires security precautions.

Reviewer: ```suggestion
1. `FromFilesSchemaProvider` loads a schema from PHP-files via `include`. That requires security precautions.
```

   Make sure you store schema file in a safe path restricted from users.
2. You can specify multiple schema files, which will be merged into one schema. 
Exception will be thrown in case of collision of roles.

3. Thanks to internal cache, loading schema from a PHP-file is so fast that you can skip an external cache at all.
But in case of loading multiple files, it may take extra time to merge them.
4. You cannot generate migrations based on PHP-file schema. [See issue #25](https://github.com/yiisoft/yii-cycle/issues/25)

> 5. Provider only reads schema. It cannot update the file after migration is applied, but `SimpleCacheSchemaProvider` does.

Reviewer: Тут идёт сравнение с SimpleCacheSchemaProvider как пример. Не уверен, но выглядит как замечание в сторону SimpleCacheSchemaProvider.

`@runtime` alias is replaced automatically. Schema will be exported into `schema.php` file.

> Make sure schema exported is correct and then switch to using it via `FromFileSchemaProvider`.

Reviewer: ```suggestion
Make sure schema exported is correct and then switch to using it via `FromFilesSchemaProvider`.
```
